Cloud Application Manager/Deploying Anywhere/using-centurylink-cloud.md

```diff
@@ -93,6 +98,12 @@ Note a couple of things about instances you deploy on CenturyLink Cloud through
 
 By default, the machine is provisioned with 17GB local disk space. You can add more disks in RAW format or Partitioned, up to 1024 GB.
 
+### Enabling services
+
+*Services* tab is the right place where services such as [Managed Services Anywhere](../Managed Services/getting-started-with-cam-enable-managed-provider.md), [Automatic Discovery of Resources](../Getting Started/register-existing-instance.md) and [Analytics](../analytics/cloudapplicationmanageranalyticsui.md) can be enabled or disabled attending to your needs. Note that the number of active services will be shown next to the tab's name.
```

Comment:
For 'Automatic Discovery of Resources', is there a section where you can link directly to?